src/test/java/io/aiven/kafka/connect/common/config/AivenCommonConfigTest.java

```diff
@@ -21,13 +21,53 @@
 import org.apache.kafka.common.config.ConfigDef;
 import org.apache.kafka.common.config.ConfigException;
 
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
 import static io.aiven.kafka.connect.common.config.AivenCommonConfig.addOutputFieldsFormatConfigGroup;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 class AivenCommonConfigTest {
 
+    private ConfigDef getBaseConfigDefinition() {
+        final ConfigDef definition = new ConfigDef();
+        addOutputFieldsFormatConfigGroup(definition, OutputFieldType.VALUE);
+
+        definition.define(AivenCommonConfig.FILE_NAME_TEMPLATE_CONFIG,
+            ConfigDef.Type.STRING, null, ConfigDef.Importance.MEDIUM,
+            "File name template", "File", 1, ConfigDef.Width.LONG,
+            "FILE_NAME_TEMPLATE_CONFIG");
+        definition.define(AivenCommonConfig.FILE_COMPRESSION_TYPE_CONFIG,
+            ConfigDef.Type.STRING, CompressionType.NONE.name, ConfigDef.Importance.MEDIUM,
+            "File compression", "File", 2, ConfigDef.Width.NONE,
+            "FILE_COMPRESSION_TYPE_CONFIG");
+
+        return definition;
+    }
+
+    @Test
+    void avroOutputFormatFilename() {
+        final Map<String, String> properties = Map.of(
+            "format.output.fields", "key,value",
+            "format.output.type", "avro"
+        );
+        final ConfigDef definition = getBaseConfigDefinition();
+
+        final AivenCommonConfig config = new AivenCommonConfig(definition, properties);
+        Assertions.assertEquals("{{topic}}-{{partition}}-{{start_offset}}.avro", config.getFilename());
```

Comment:
should we stick to `assertj` assertions? e.g. `assertThat(config.getFilename()).isEqualTo(...)`

Reply:
Sure.